### PR TITLE
Saliency map via guided backpropagation implemented

### PR DESCRIPTION
**Pull request by rfratila**
Working on the tests now but want you guys to look it over while I do that.

### CODE REVIEW

**Comment by priyatharsan** [comment]
I agree with adding `utils` folder.

File: vulcanai2/plotters/visualization.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 from datetime import datetime
 
+from .utils import GuidedBackprop
+
```
Comment (priyatharsan) [inline vulcanai2/plotters/visualization.py]: I don't see the `utils.py`, are you planning to add it as `utils/plotters.py`?

**Comment by priyatharsan** [comment]
looks good to me 👍

File: vulcanai2/plotters/visualization.py (hunk 1 above)
Comment (rfratila) [reply vulcanai2/plotters/visualization.py]: yeah I plan on doing the folder restructure after I'm done

**Comment by rfratila** [comment]
I found a weird bug with nested activations that have different names `activation` and `_activation`

**Comment by priyatharsan** [comment]
I think @nirtiac removed them in the recent push?

**Comment by nirtiac** [comment]
> it got me thinking, we should have a utils folder in our models, plotters, etc level so we have a consistent way of finding utility functions e.g. `from vulcanai.utils.plotters import *` instead of what we have now were we need to do `from vulcanai.plotters.utils import *`. Thoughts @priyatharsan @nirtiac ?

I don't see the issue? it's just the relevant model plus utils, it's just a different pattern and not more consistent, no?

File: vulcanai2/models/metrics.py [hunk 2]
```diff
@@ -212,29 +144,29 @@ def run_test(self, model, test_x, test_y, figure_path=None, plot=False):
         print ('\nAccuracy: {}'.format(accuracy))
 
         print ('Sensitivity:'),
-        print(self.round_list(sens, decimals=3))
+        print(round_list(sens, decimals=3))
```
[the remark this answers is not in the document]
Comment (rfratila) [reply vulcanai2/models/metrics.py]: yeah I think that's a good idea

**Pull request merged**